sql/information_schema/information_schema.go

```diff
@@ -419,6 +419,7 @@ var partitionSchema = Schema{
 	{Name: "partition_expression", Type: MustCreateStringWithDefaults(sqltypes.VarChar, 2048), Default: nil, Nullable: true, Source: PartitionsTableName},
 	{Name: "subpartition_expression", Type: MustCreateStringWithDefaults(sqltypes.VarChar, 2048), Default: nil, Nullable: true, Source: PartitionsTableName},
 	{Name: "partition_descriptor", Type: LongText, Default: nil, Nullable: true, Source: PartitionsTableName},
```

Comment:
This is actually just a typo: there is no partition_descriptor column, just the one you added.